actions/user_actions.test.js

```diff
@@ -21,6 +26,19 @@ jest.mock('mattermost-redux/actions/users', () => {
     };
 });
 
+jest.mock('mattermost-redux/selectors/entities/channel_categories', () => {
```

Comment:
In particular, why mock these selectors?

Reply:
If we are calling the selectors then isn't that a good enough test? Selectors are supposed to filter and they have tests to cover that.

Reply:
The main thing I'm concerned about is that the tests confirm that the code call the selectors, but it doesn't confirm that calling the selectors actually prevents the extra requests. In the end, they both cover the same thing assuming that the selectors work as expected, so while I personally think that testing for the bug would be better, doing it this way is probably fine as well, so 0/5 on whether or not we change it.